src/main/java/uk/gov/companieshouse/api/accounts/util/ixbrl/accountsbuilder/SmallFullAccountsBuilderImpl.java

```diff
@@ -0,0 +1,80 @@
+package uk.gov.companieshouse.api.accounts.util.ixbrl.accountsbuilder;
+
+import java.io.IOException;
+import org.springframework.stereotype.Component;
+import uk.gov.companieshouse.api.accounts.model.ixbrl.Account;
+import uk.gov.companieshouse.api.accounts.model.ixbrl.balancesheet.BalanceSheet;
+import uk.gov.companieshouse.api.accounts.model.ixbrl.balancesheet.CalledUpSharedCapitalNotPaid;
+import uk.gov.companieshouse.api.accounts.model.ixbrl.company.Company;
+import uk.gov.companieshouse.api.accounts.model.ixbrl.notes.Notes;
+import uk.gov.companieshouse.api.accounts.model.ixbrl.notes.PostBalanceSheetEvents;
+import uk.gov.companieshouse.api.accounts.model.ixbrl.period.Period;
+
+@Component
+public class SmallFullAccountsBuilderImpl implements AccountsBuilder {
+
+    @Override
+    public Object buildAccount() throws IOException {
+        return getSmallFullAccount();
+    }
+
+    /**
+     * Builds the small full accounts model. Functionality needs to be change.
```

Comment:
This comment will be confusing to developers in the future who have not been involved. Please reference Jira tickets explaining the changes.

Reply:
4a47945